docker/docker/src/test/java/org/arquillian/cube/docker/impl/client/containerobject/CubeContainerObjectTestEnricherTest.java

```diff
@@ -13,10 +12,14 @@
 import static org.mockito.Mockito.when;
 
 import java.io.File;
-import java.io.FilenameFilter;
-import java.util.ArrayList;
+import java.io.IOException;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+
+import com.google.common.base.StandardSystemProperty;
```

Comment:
I would not suggest to use Guava here. The main idea is that this dependency is transitive, this means that maybe in the future is not bundled anymore because a third library just removes its usage.

Also I would not add it as "real" dependency on project because it is a 3MB library for just using one class.